src/modules/service-catalog/components/useItemFormFields.tsx

```diff
@@ -87,14 +102,17 @@ export function useItemFormFields(
   baseLocale: string
 ) {
   const [requestFields, setRequestFields] = useState<Field[]>([]);
+  const [associatedLookupField, setAssociatedLookupField] =
+    useState<Field | null>();
 
   useEffect(() => {
     const fetchAndSetFields = async () => {
       if (serviceCatalogItem && serviceCatalogItem.form_id) {
         try {
-          await fetchTicketFields(serviceCatalogItem.form_id, baseLocale).then(
-            (ticketFields) => setRequestFields(ticketFields)
-          );
+          const { requestFields, associatedLookupField } =
+            await fetchTicketFields(serviceCatalogItem.form_id, baseLocale);
+          setRequestFields(requestFields);
+          setAssociatedLookupField(associatedLookupField);
```

Comment:
We should save the error state somewhere if don't find the lookup field. We can tackle this separately when we are going to handle the error state, the important thing is that we don't forget to do it